sdk/core/azure-core/azure/core/pipeline/policies/_distributed_tracing.py

```diff
@@ -88,6 +89,8 @@ def on_request(self, request):
             span_name = namer(request.http_request)
 
             span = span_impl_type(name=span_name)
+            for attr, value in self._tracing_attributes.items():
```

Comment:
Are the `tracing_attributes` only supposed to be provided in `__init__`? (i.e. not on a per-request basis?)

Reply:
I think that's enough, but it's open design at this point :). What you feel?

Reply:
I guess the caller can always add attributes to the current (ambient) span directly if they had a need to do so (e.g. add a client request id or something similar). 

And if we do want to support the addition of per-request level parameters, it is an additive change.